src/uplift/bot/requirements.nix

```diff
@@ -1289,6 +1289,7 @@ let
       self."more-itertools"
       self."pluggy"
       self."py"
+      self."requests"
```

Comment:
I think this creates a circular dependency, in https://github.com/mozilla/release-services/pull/1695/files#diff-8b78f6731647511472cecf6e0bd0bf40R195 I've overrode it, but #1695 isn't merged yet.

Reply:
I just removed the pytest update with the circular dep